### PR TITLE
feat: transaction builder

### PR DESCRIPTION
**Pull request by 0xArdy**

### CODE REVIEW

File: src/containers/BridgeCard/CustomTransaction.tsx [hunk 1]
```diff
@@ -68,15 +61,40 @@ export const CustomTransaction = () => {
...
-              modal={false}
-              placeholder='Enter custom data'
-            />
+                {selectedFunction &&
+                  selectedFunction.inputs.map((input: AbiParameter, index) => (
+                    <InputField
+                      key={input.type + input?.name + index}
+                      label={input.name}
+                      value={functionParams?.[input.name || ''] || ''}
+                      setValue={(val) => setFunctionParams({ ...functionParams, [input.name || '']: val })}
```
Comment (turtlemoji) [inline src/containers/BridgeCard/CustomTransaction.tsx:91]: maybe put this values on a const up to you

File: src/containers/Modals/Error.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        Confirm
+      </PrimaryButton>
+    </BaseModal>
+  );
+};
+
+const ModalBody = styled(Box)(() => {
+  const { currentTheme } = useCustomTheme();
+  return {
+    marginTop: '-3.2rem',
```
Comment (turtlemoji) [inline src/containers/Modals/Error.tsx:31]: AAAAAAAAAAAAahhhhHHHhhhhhhhhhhhhhhhh

**Review by turtlemoji — APPROVED** [review]
pre-approved lgtm

File: src/containers/AccountHistory/Table.tsx [hunk 3]
```diff
@@ -135,25 +156,35 @@ const STableBody = styled(TableBody)(() => {
...
       width: '6rem',
     },
+
+    '.row-ready-to-prove, .row-ready-to-finalize': {
+      backgroundColor: '#231710',
+    },
+    '.row-ready-to-prove:hover, .row-ready-to-finalize:hover': {
+      filter: 'brightness(1.1)',
+      backgroundColor: '#231710',
+    },
```
[the remark this answers is not in the document]
Comment (0xArdy) [reply src/containers/AccountHistory/Table.tsx:166]: ill do it in the next pr :+1:

[the remark this answers is not in the document]
Comment (0xArdy) [reply src/containers/AccountHistory/Table.tsx:166]: https://github.com/defi-wonderland/superchain-bridge-ui/pull/36/commits/5ac22d369e2bf72d87a62c1214d3621d47bc3d8a

File: src/containers/Modals/Error.tsx (hunk 2 above)
Comment (0xArdy) [reply src/containers/Modals/Error.tsx:31]: same issue as success modal component :T

**Pull request merged**